elevate-cpanel

```diff
@@ -249,21 +251,29 @@ use constant VETTED_YUM_REPO => qw{
   CentOS-Sources.repo
   CentOS-Vault.repo
   CentOS-x86_64-kernel.repo
+  centos-extras.repo
```

Comment:
How are we re-installing these repos? does leapp handle them already?

Reply:
Yes, it provides CentOS 8 Extras and AlmaLinux 8 Extras to upgrade packages from CentOS 7 Extras.